src/core/Akka.FSharp/Spawning.fs

```diff
@@ -0,0 +1,140 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Spawning.fs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Akka.FSharp
+
+open Akka.Actor
+open System
+open Microsoft.FSharp.Quotations
+open Microsoft.FSharp.Linq.QuotationEvaluation
+
+[<RequireQualifiedAccess>]
+module Configuration = 
+    /// Parses provided HOCON string into a valid Akka configuration object.
+    let parse = Akka.Configuration.ConfigurationFactory.ParseString
+    
+    /// Returns default Akka configuration.
+    let defaultConfig = Akka.Configuration.ConfigurationFactory.Default
+    
+    /// Loads Akka configuration from the project's .config file.
+    let load = Akka.Configuration.ConfigurationFactory.Load
+
+module System = 
+    /// Creates an actor system with remote deployment serialization enabled.
+    let create (name : string) (config : Akka.Configuration.Config) : ActorSystem = 
+        let system = ActorSystem.Create(name, config)
+        Serialization.exprSerializationSupport system
+        system
+
+[<AutoOpen>]
+module Spawn = 
```

Comment:
I agree, definitely, with the partial application perspective. I was thinking that options was more general, though. Similar to `Props` creation, this makes it more amenable to having a partially applied:

    let createWithDefaultPool = spawnOpt system [SpawnOption.Router (RoundRobinPool(10))]

I thought that it was more natural for such purposes, but this is of course easily done by consumers if they want.

On the naming, my recommendation would be for `[<RequireQualifiedAccess>]`, but I wholly understand keeping things more in line with the Erlang and Scala roots.